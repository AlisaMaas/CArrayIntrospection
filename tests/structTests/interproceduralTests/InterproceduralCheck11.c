#include "structHeader.h"

/**
* Fixed length check - should find
* that fixed is fixed length.
**/

int foo (struct baz *b) 
{
    int sum = 0;
    for (int i = 0; i < 30; i++) 
    {
        sum += (int) b->fixed[i];
    }
    return sum;
}