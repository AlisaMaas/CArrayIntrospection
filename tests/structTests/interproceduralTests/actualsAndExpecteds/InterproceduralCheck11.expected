struct struct.baz[0] should be annotated Fixed length of 30.
